forui/lib/src/widgets/avatar/avatar.dart

```diff
@@ -22,25 +22,25 @@ class FAvatar extends StatelessWidget {
   /// The circle's size.
   final double size;
 
-  /// The fallback widget displayed if image parameter fails to load.
+  /// The child, typically an image.
   ///
-  /// Typically used to display the user's initials using a [Text] widget
-  /// styled with [FAvatarStyle.backgroundColor].
-  ///
-  /// Use image parameter to display an image; use [fallback] for initials.
-  final Widget fallback;
+  /// If the image fails to load, a fallback widget, typically a [Text] widget used to display the user's initials
+  /// styled with [FAvatarStyle.backgroundColor] is displayed.
```

Comment:
Should this be documented here. Might be better to place it in the constructor since it's called a fallback there and the child widget here doesn't really do that as well.